pkg/git/git_test.go

```diff
@@ -113,6 +113,14 @@ func TestValidateGitSSHURLFormat(t *testing.T) {
 			url:  "user@host.xz:path/to/repo.git",
 			want: true,
 		},
+		{
+			url:  "http://github.com/user/project.git",
+			want: false,
+		},
+		{
+			url:  "https://github.com:443/user/project.git",
+			want: false,
+		},
```

Comment:
Maybe add a case with a different port than 443? So it will be aligned with your issue